data/announcements/scheduled.yaml

```
@@ -71,4 +71,26 @@ announcements:
     <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-europe/">KubeCon + CloudNativeCon EU 2021</a> <em>virtual</em>.
   message: |
     4 days of incredible opportunities to collaborate, learn + share with the entire community!<br />
-    May 4 - May 7, 2021.
\ No newline at end of file
+    May 4 - May 7, 2021.
+
+- name: Kubecon 2021 NA
+  startTime: 2021-10-01T00:00:00
+  endTime: 2021-10-15T00:00:00
+  style: "background: linear-gradient(90deg, rgb(7, 132, 111) 0%, rgb(54, 214, 183) 100%)" 
+  title: |
+    <img src="/images/announcements/kubecon-China-2021-white.svg"float: right; height: 80px;" />
+    <a href="https://events.linuxfoundation.org/kubecon-cloudnativecon-north-america/"> KubeCon + CloudNativeCon North America 2021</a> <em>Los Angeles, California + Virtual</em>.
+  message: |
+    5 days of incredible opportunites to collaborate, learn + share with entire community!<br />
+    October 11 - 15, 2021.
+
+- name: Kubecon 2021 China
+  startTime: 2021-11-30T00:00:00
+  endTime: 2021-12-10T00:00:00
```

The site only updates its banner a couple of times a day but, to ensure it stays shown whilst the conference is running:
```suggestion
  endTime: 2021-12-10T14:00:00
```